### PR TITLE
helm: add extraDeploy option

### PR DESCRIPTION
**Pull request by mustdiechik**
To deploy additional manifests with the release.

### CODE REVIEW

**Comment by nixpanic** [comment]
Hi @mustdiechik , thanks for the PR!

Could you explain a little how this can be used, what advantage does it have? I am not very familiar with Helm common practices, so a pointer to this recommendation would help me.

Please also document the new option in the `docs/deploy-*.md` files, so that other users are aware of it.

**Comment by mustdiechik** [comment]
with this options i deploy additional PVC because I have few pools (ssd\hdd).
I will use this option to deploy calico rules for security purpose.
It is best practice to have this functionality in chart to avoid construct umbrella chart to use for deploying additional manifest.

**Comment by nixpanic** [comment]
> with this options i deploy additional PVC because I have few pools (ssd\hdd). I will use this option to deploy calico rules for security purpose. It is best practice to have this functionality in chart to avoid construct umbrella chart to use for deploying additional manifest.

Thanks for explaining, that sounds reasonable to me. Could you add that to the documentation, so that other users are aware of the option?

File: docs/deploy-cephfs.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ you're running it inside a k8s cluster and find the config itself).
 | `csi.storage.k8s.io/provisioner-secret-namespace`, `csi.storage.k8s.io/node-stage-secret-namespace` | for Kubernetes | Namespaces of the above Secret objects                                                                                                                                                                                  |
 | `encrypted`                                                                                         | no             | disabled by default, use `"true"` to enable fscrypt encryption on PVC and `"false"` to disable it. **Do not change for existing storageclasses**                                                                          |
 | `encryptionKMSID`                                                                                   | no             | required if encryption is enabled and a kms is used to store passphrases                                                                                                                                                |
-
+| `extraDeploy`                                                                                       | no             | array of extra objects to deploy with the release |
```
Comment (nixpanic) [inline docs/deploy-cephfs.md]: I guess you need to keep an empty line after the table. The MarkDown linter complains about the table, but that shouldn't be the issue.

Comment (mustdiechik) [reply docs/deploy-cephfs.md]: Thx, fixed

**Review by nixpanic — APPROVED** [review]

**Review by riya-singhal31 — APPROVED** [review]

**Comment by nixpanic** [comment]
@mergifyio rebase

**Comment by nixpanic** [comment]
@mergifyio queue

**Pull request merged**